.circleci/config.yml

```
@@ -234,28 +180,18 @@ jobs:
               --arg collectCoverage true \
               --attr doc
 
```

(via https://circleci.com/blog/managing-secrets-when-you-have-pull-requests-from-outside-contributors/)
```suggestion

      - run:
          name: Early return if this build is from a forked PR
          command: |
            if [ -n "$CIRCLE_PR_NUMBER" ]; then
              echo "Nothing to do for forked PRs, so marking this step successful"
              circleci step halt
            fi
            
```

I guess this and the push need to move after the coverage reporting, though.